tests/test_register.py

```diff
@@ -25,7 +25,8 @@ def register_settings(make_settings):
 
 
 def test_successful_register(register_settings):
-    """Test a successful package registration"""
+    """A package is successfully registered when a valid repository
```

Comment:
Same thing here. Further, the first line should fit on one-line, including a period and additional information should be specified after a blank line in paragraph form.

Reply:
For now, I have gone with one-line docstrings terse enough to make flake8 happy. Do let me know if they are too terse and should be done in a multi-line format.